src/pages/AdoptablePetsDetails.jsx

```diff
@@ -1,18 +1,19 @@
 import React from "react";
```

Comment:
Do we anticipate having a pet details page for each status? (lost, reported lost, adoptable)?

Reply:
@martypowell hadnt really considered that but it would make sense. Should we change this to SelectedPetDetails or something more generic along those lines?

Reply:
Pet?

Reply:
done